devel/d3dmetal/Portfile

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                d3dmetal
+version             1.1
+epoch               0
+categories          devel
+maintainers         {@Gcenx gmail.com:gcenx83}
+homepage            https://developer.apple.com/games
+platforms           {darwin any >= 23}
```

Comment:
`any` means no architecture-specific software is installed; this is contradicted by the subsequent line `supported_archs x86_64`.

Reply:
Oh, ok yes you're right.